core/src/main/java/io/questdb/std/str/StringSink.java

```diff
@@ -25,10 +25,11 @@
 package io.questdb.std.str;
 
 import io.questdb.std.Chars;
+import io.questdb.std.DeepCloneable;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-public class StringSink implements MutableUtf16Sink, CharSequence, CloneableMutable, Utf16Sink {
+public class StringSink implements MutableUtf16Sink, CharSequence, CloneableMutable, Utf16Sink, DeepCloneable<StringSink> {
```

Comment:
`Utf8StringSink` (and maybe some other sinks that can be met in `Function`s) should be also cloneable.